frontend/src/components/ShopTicketList/ShopTicketList.jsx

```diff
@@ -29,17 +29,22 @@ const ticketTypes = [
   },
 ];
 
-const TicketList = () => {
+const ShopTicketList = () => {
   return (
     <div className="ticket-list-container">
       <div className="ticket-list">
         <h1 className="main-title">Ticket Types</h1>
-        {ticketTypes.map(i => (
-          <AdminTicket key={i.id} {...i} />
+        {shopTicketTypes.map(shopTicket => (
+          <Ticket
+            key={shopTicket.id}
+            {...shopTicket}
+            canEdit={false}
+            buttonText="BUY"
+          />
```

Comment:
@marcianszlovak maybe we should have the buttons as children elements:
```suggestion
          <Ticket
            key={shopTicket.id}
            {...shopTicket}
          >
          <Button buy />
          <Button edit />
          <...>
          </Ticket>
```

Reply:
@marcianszlovak don't put the button text and action into the mock data, just use children components in the `Ticket` component